src/lib/eth-calls.ts

```diff
@@ -12,12 +12,9 @@ async function registerTable(
   const address = await signer.getAddress();
   /* eslint-disable-next-line camelcase */
 
-  const network = this.network;
+  const contractAddress = this.contract ?? contractAddresses[this.network];
```

Comment:
The meat of this PR is here.  As a developer of the SDK or the SC you don't need to add the SC address to the [util.ts](https://github.com/tablelandnetwork/js-tableland/blob/joe/contract-op/src/lib/util.ts) file anymore.  You can pass it in a connection option